rllib/utils/spaces/tuple_actions.py

```diff
@@ -0,0 +1,11 @@
+from collections import namedtuple
```

Comment:
Please don't add new packages in unrelated PRs. Let's just put this file utils (or keep in existing location).